internal/modelcontroller/engine_vllm.go

```diff
@@ -27,24 +28,19 @@ func (r *ModelReconciler) vLLMPodForModel(m *kubeaiv1.Model, c ModelConfig) *cor
 		"--model=" + vllmModelFlag,
 		"--served-model-name=" + m.Name,
 	}
+	if m.Spec.Adapters != nil {
+		args = append(args, "--enable-lora")
+	}
 	args = append(args, m.Spec.Args...)
 
 	env := []corev1.EnvVar{
 		{
-			// TODO: Conditionally set this token based on whether
-			// huggingface is the model source.
-			Name: "HF_TOKEN",
-			ValueFrom: &corev1.EnvVarSource{
-				SecretKeyRef: &corev1.SecretKeySelector{
-					LocalObjectReference: corev1.LocalObjectReference{
-						Name: r.HuggingfaceSecretName,
-					},
-					Key:      "token",
-					Optional: ptr.To(true),
-				},
-			},
+			// https://docs.vllm.ai/en/latest/models/lora.html#dynamically-serving-lora-adapters
+			Name:  "VLLM_ALLOW_RUNTIME_LORA_UPDATING",
```

Comment:
I prefer to make this conditional to only be enabled when adapters are set like you did with `--enable-lora` flag

Reply:
Done